README-cdk.md

```diff
@@ -120,8 +120,8 @@ Verify that `kubefed2` is working:
 Clone the demo code to your local machine:
 
 ~~~sh
-    git clone https://github.com/openshift/federation-dev.git
-    cd federation-dev
+git clone https://github.com/openshift/federation-dev.git /var/tmp/federation-dev/
```

Comment:
I'm not sure we want to "force" `/var/tmp` here... beyond individual preferences, how well does that work cross-platform?

Reply:
As long as we are running on a distribution following the standard [LFSH](https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard) it should work. We can move it to /tmp/ or any other path.

The thing here is that if the user following this guide changes to another directory, using relative paths could be a problem.